worker/resumer/manifold.go

```diff
@@ -36,7 +35,7 @@ func (config ManifoldConfig) newWorker(a agent.Agent, apiCaller base.APICaller)
 	// This bit should be encapsulated in another manifold
 	// satisfying jujud/agent/engine.Flag, as described in
 	// the implementation below. Shouldn't be a concern here.
-	if ok, err := isModelManager(a, apiCaller); err != nil {
```

Comment:
should we still need to check if it's managing model or not

Reply:
We call IsController() instead